NiKomLib/smakefile

```diff
@@ -4,78 +4,80 @@ OBJS    = LibInit.o UserLibInit.o Matrix.o Rexx.o Lock.o ReadFidoText.o Echo.o \
 SCFLAGS = STREQ STRINGMERGE NOSTKCHK LIBCODE UNSIGNEDCHAR IDIR=/Include
 DEBUG   = DEBUG=LINE
 
-all :  nikom_pragmas.h nikom.library
+all:    nikom_pragmas.h nikom.library
 
-.c.o :
-	sc $(SCFLAGS) $(DEBUG) $*
+.c.o:
+        sc $(SCFLAGS) $(DEBUG) $*
 
-setup :
-	makedir Debug
-	makedir NoDebug
+debug:
+  makedir debug
 
-clean :
-   delete \#?.o nikom_pragmas.h
-   delete Debug/\#? NoDebug/\#?
+nodebug:
+  makedir nodebug
 
-nikom_pragmas.h : nikom.fd
-		fd2pragma	nikom.fd nikom_pragmas.tmp
-		rx convpragma nikom_pragmas.tmp nikom_pragmas.h
-		copy nikom_pragmas.h /Include
-		delete nikom_pragmas.tmp
+clean:
+  delete \#?.o nikom_pragmas.h
+  delete debug nodebug all
 
-nikom.library : $(OBJS) nikom.fd
-   slink TO Debug/nikom.library NOICONS \
-         FROM lib:libent.o $(OBJS) \
-         LIBPREFIX _LIB LIBFD nikom.fd \
-         LIB lib:sc.lib lib:amiga.lib \
-         BATCH NOALVS SMALLCODE SMALLDATA \
-         LIBVERSION 21 \
-         LIBREVISION 1 \
-         LIBID "nikom.library 21.1"
-	slink Debug/nikom.library TO NoDebug/nikom.library NODEBUG NOICONS
+nikom_pragmas.h: nikom.fd
+  fd2pragma nikom.fd nikom_pragmas.tmp
+  rx convpragma nikom_pragmas.tmp nikom_pragmas.h
+  copy nikom_pragmas.h /Include
+  delete nikom_pragmas.tmp
 
-LibInit.o      : LibInit.c      nikom.fd
+nikom.library : debug nodebug $(OBJS) nikom.fd
+  slink TO Debug/nikom.library NOICONS \
+        FROM lib:libent.o $(OBJS) \
+        LIBPREFIX _LIB LIBFD nikom.fd \
+        LIB lib:sc.lib lib:amiga.lib \
+        BATCH SMALLCODE SMALLDATA \
+        LIBVERSION 21 \
+        LIBREVISION 1 \
+        LIBID "nikom.library 21.1"
+  slink Debug/nikom.library TO NoDebug/nikom.library NODEBUG NOICONS
 
-UserLibInit.o  : UserLibInit.c  nikom.fd CharTabs.h
+LibInit.o:      LibInit.c       nikom.fd
 
-Matrix.o       : Matrix.c       nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
+UserLibInit.o:  UserLibInit.c   nikom.fd CharTabs.h
 
-Rexx.o         : Rexx.c         nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
+Matrix.o:       Matrix.c        nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
 
-Lock.o         : Lock.c         nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
+Rexx.o:         Rexx.c          nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
 
-ReadFidoText.o : ReadFidoText.c nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
+Lock.o:         Lock.c          nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
 
-Echo.o         : Echo.c         nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
+ReadFidoText.o: ReadFidoText.c  nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
 
-NodeTypes.o    : NodeTypes.c    nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
+Echo.o:	        Echo.c          nikom.fd /Include/NiKomLib.h /Include/NiKomStr.h
```

Comment:
Weird.. look at the "Files changed" tab in the pull request (and use the split view, not the unified). Then it doesn't line up for me. But when looking directly at your branch it does line up. Let's see how it looks when it's merged. :-)